doc/tags-ja

```diff
@@ -0,0 +1,9 @@
+!_TAG_FILE_ENCODING	utf-8	//
+b:highlightedyank_highlight_duration	highlightedyank.jax	/*b:highlightedyank_highlight_duration*
+g:highlightedyank_highlight_duration	highlightedyank.jax	/*g:highlightedyank_highlight_duration*
+highlightedyank-highlight-groups	highlightedyank.jax	/*highlightedyank-highlight-groups*
+highlightedyank-index	highlightedyank.jax	/*highlightedyank-index*
+highlightedyank-introduction	highlightedyank.jax	/*highlightedyank-introduction*
+highlightedyank.txt	highlightedyank.jax	/*highlightedyank.txt*
+highlightedyank.vim	highlightedyank.jax	/*highlightedyank.vim*
+hl-HighlightedyankRegion	highlightedyank.jax	/*hl-HighlightedyankRegion*
```

Comment:
It may duplicate in other's environment, it is safer not to include help tag file. Could you omit this file?